client/packages/invoices/src/InboundShipment/DetailView/modals/utils.ts

```diff
@@ -32,6 +32,7 @@ const createDraftInboundLine = ({
     location: undefined,
     type,
     item,
+    itemName: item.name,
```

Comment:
So this was not populated at all previously?

Reply:
Nope... some of the item related stuff aren't populated in most of the drafts.